style.css

```diff
@@ -471,3 +479,396 @@ h2 {
+    padding-bottom: 30px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url('./Images/Icon\ -\ Mail.png');
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  .close {
+    display: none;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url("./Images/illustration-header2.png");
+    background-size: 800px 968px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: -60px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 153px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    margin-top: 150px;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 185px;
+    margin-top: 70px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    background-repeat: no-repeat;
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
```

Comment:
- [ ] **[OPTIONAL]** Please make sure the content of the work section doesn't go outside of the screen when the screen is resized. You can do that by taking advantage of flex-box features (`flex-wrap: wrap;`) for the part that uses flex and updating the grid-template depending on the screen size. 

**When the project is resized**
![image](https://user-images.githubusercontent.com/74411135/236287681-1ad43574-7420-4a17-a54c-01668ab8445c.png)

This will make your project way accessible for different screen sizes which will improve the user experience 📱  😄 👍